packages/vue/src/components/molecules/SfSearchBar/SfSearchBar.vue

```diff
@@ -12,18 +12,24 @@
       @blur="$emit('blur')"
     />
     <!-- @slot -->
-    <slot name="icon">
-      <span v-if="icon" class="sf-search-bar__icon">
-        <SfIcon :color="icon.color" :size="icon.size" icon="search" />
-      </span>
-    </slot>
+    <SfButton
```

Comment:
Shouldn't this button be inside the icon slot? :slightly_smiling_face:

Reply:
You are right, I changed that.